### ISSUE TITLE
RFC compliance - CNAMETests test_ptr_exists and test_ptr_cname_exists

### ISSUE THREAD

**Issue by murrown** [OSU-Net/cyder#8]
To reproduce: Run CNAMETests
These tests ensure that a PTR record does not point to a CNAME, in compliance with RFC 1034 (maybe 1912?). There /are/ checks in place in the PTR and CNAME models, but this code has been commented out because there are/were preexisting records in Maintain that violate this rule. We need to verify that it is safe to reintroduce this code so that these tests will pass.

In other words, if we reintroduce the code and attempt a migration, and everything goes fine, then we're good to go.
